src/battle-dex.ts

```diff
@@ -424,7 +424,7 @@ const Dex = new class implements ModdedDex {
 		}
 		if (!table) return pokemon.tier;
 
-		let id = pokemon.id;
+		let id = toID(pokemon.species);
```

Comment:
Can you clarify which pokemon for which `pokemon.id` doesn't match `toID(pokemon.species)`?

Reply:
`id` didn't exist on the pokemon template that was being passed to getTier. That was what was causing this problem. This is a quick fix to get the Double's teambuilder working again until I can investigate more and find out why.

Reply:
Revert?

Reply:
Oh, I see. Yeah, this works.